hdr_html_canvas_element.md

```diff
@@ -301,33 +323,30 @@ ITU-R BT.2408-5, Clause 6](https://www.itu.int/pub/R-REP-BT.2408)
 
 Tone mapping from `rec2100-pq` to `srgb` is performed using the following steps:
 
-* apply the Parametric Tone Mapping Method specified at [SMPTE ST 2094-10, Annex
-B](https://ieeexplore.ieee.org/document/9405553) using the following recommended
-parameter values:
-  * `TargetedSystemDisplayMaximumLuminance` = 100
-  * `TargetedSystemDisplayMinimumLuminance` = 0.1
-* convert to sRGB using `rec2020SDRLineartoSRGB()` below
+* apply the EETF specified at [Rep. ITU-R BT.2408, Annex
+5](https://www.itu.int/pub/R-REP-BT.2408) using the following parameter values:
+  * L<sub>B</sub> = `CanvasColorVolumeMetadata::minimumLuminance` || 0;
+  * L<sub>W</sub> = `CanvasColorVolumeMetadata::maximumLuminance` || 1000;
+  * L<sub>min</sub> = 0
+  * L<sub>max</sub> = 203
+* convert to sRGB using `rec2100PQtoSRGB()` below
 
 ```javascript
-function rec2020SDRLineartoSRGB(r, g, b) {
-  const [rt, gt, bt] = matrixXYZtoRec709(matrixBT2020toXYZ(r, g, b));
-  const rp = Math.pow(rt / 100, 1/2.4);
-  const gp = Math.pow(gt / 100, 1/2.4);
-  const bp = Math.pow(bt / 100, 1/2.4);
+function rec2100PQtoSRGB(r, g, b) {
+  let rt = 10000 * pqEOTF(r) / 203;
+  let gt = 10000 * pqEOTF(g) / 203;
+  let bt = 10000 * pqEOTF(b) / 203; 
+  [rt, gt, bt] = matrixXYZtoRec709(matrixBT2020toXYZ(rt, gt, bt));
+  const rp = Math.pow(rt, 1/2.4);
+  const gp = Math.pow(gt, 1/2.4);
+  const bp = Math.pow(bt, 1/2.4);
   return [rp, gp, bp];
 }
```

Comment:
Btw. this seems to be a direct colorimetric conversion, like a change of basis in linear algebra, rather than a color gamut mapping. You will get out-of-range sRGB values, which I presume will then be hard-clipped independently on each color channel. This will cause loss of detail and saturation.

What is the value of this example? I doubt it would ever be used in practice as-is.

There is also a mathematical problem, because a negative value raised to fractional power results in a complex number, unless `matrixXYZtoRec709()` does clipping internally?

Curiously the demo link to sandflow below is using an almost achromatic example image which would not exhibit color gamut problems.

Reply:
Good point, this is indeed a colorimetric conversion. 

In CSS Color 4, all the predefined RGB spaces are defined over the extended range. We don't support 709 but do s[upport  sRGB over the extended range](https://drafts.csswg.org/css-color-4/#predefined-sRGB), as an example.

If had clipping is expected, then the stage at which it occurs should be specified and the choice of hard clip (with associated lightness and hue changes) justified.

Reply:
> What is the value of this example? I doubt it would ever be used in practice as-is.

The value of this example is to demonstrate tone-mapping from a high dynamic range image to a narrow dynamic range image, where high and narrow dynamic range refer to luminance range. It does not demonstrate mapping from a wide color range to a narrow color range, which is not a new problem (last I checked many monitors could not display the full sRGB/Rec. 709 gamut).